src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: $widthCalendar;
+  &__day {
+    width: $widthDay;
+    height: $heightDay;
+    margin: $marginBetweenDays / 2;
+    box-sizing: border-box;
+    background-color: $backgroundDay;
+    border: 1px solid #000;
+    position: relative;
+    transition: all 0.5s ease 0s;
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-50px);
```

Comment:
The transform property on line 18 uses translateY(-50px), which might be too large for a hover effect. Consider using a smaller value for a smoother transition.